src/App.jsx

```diff
@@ -13,71 +15,69 @@ export const goods = [
   'Jam',
   'Garlic',
 ];
+const NO_GOODS_MESSAGE = 'No goods selected';
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? (
+            <>
+              {`${selectedGood} is selected`}
+              <button
+                onClick={() => setSelectedGood('')}
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </>
+          )
+          : NO_GOODS_MESSAGE}
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = good === selectedGood;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+            return (
+              <tr
+                key={good}
+                data-cy="Good"
+                className={cn(
+                  { 'has-background-success-light': isSelected },
+                )}
+              >
+                <td>
+                  <button
+                    onClick={() => {
+                      setSelectedGood(!isSelected && good);
+                    }}
+                    data-cy={
+                      isSelected
+                        ? 'RemoveButton'
+                        : 'AddButton'
+                    }
+                    type="button"
+                    className={cn(
+                      'button', { 'is-info': isSelected },
+                    )}
+                  >
+                    {isSelected ? '-' : '+'}
```

Comment:
I think it's better just to render two different buttons to improve the readability